src/test/javascript/spec/component/iris/exercise-chat-widget.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ExerciseChatWidgetComponent } from 'app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component';
+import { ChatbotPopupComponent } from 'app/overview/exercise-chatbot/chatbot-popup/chatbot-popup.component';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { FormsModule } from '@angular/forms';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { MockPipe } from 'ng-mocks';
+import { MatDialogModule } from '@angular/material/dialog';
+
+describe('ExerciseChatWidgetComponent', () => {
+    let component: ExerciseChatWidgetComponent;
+    let fixture: ComponentFixture<ExerciseChatWidgetComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [FormsModule, FontAwesomeModule, MatDialogModule],
+            declarations: [ExerciseChatWidgetComponent, ChatbotPopupComponent, MockPipe(ArtemisTranslatePipe)],
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
```

Comment:
You don't need the second `beforeEach` block. Just add it after then compileComponents call like this: 

```ts
compileComponents().then(() => {
    // Your code
});

Reply:
Done